lite/examples/image_classification/ios/ImageClassification/ViewControllers/InferenceViewController.swift

```diff
@@ -65,7 +65,7 @@ class InferenceViewController: UIViewController {
   private let bottomSpacing: CGFloat = 21.0
   private let minThreadCount = 1
   private let bottomSheetButtonDisplayHeight: CGFloat = 44.0
-  private let infoTextColor = UIColor.black
+  private let infoTextColor = UIColor(named: "darkOrLight")
```

Comment:
Thanks for the fix!

This API only available from iOS 11 but we need to support older iOS versions as well. Could you change it so that UIColor.black is used in older iOS versions?

For your reference:
https://www.hackingwithswift.com/new-syntax-swift-2-availability-checking

Reply:
Ah, you were faster than me :)
I have some more concrete code suggestions in my own comment. Please take a look.

Reply:
Completed. Thanks for the review.